lib/win32/certstore/mixin/crypto.rb

```diff
@@ -122,6 +127,24 @@ def initialize(str = nil)
           end
         end
 
+        class CRYPT_HASH_BLOB < FFI::Struct
+          layout :cbData, DWORD, # Count, in bytes, of data
+                 :pbData, :pointer # Pointer to data buffer
+
+          def initialize(str = nil)
+            super(nil)
+            if str
+              # Converting the string to its byte array
+              arr = [str].pack("H*").unpack("C*")
```

Comment:
we can name variable itself to something byte_arr

Reply:
Have made the changes. Thanks